add-chain/cmd/compress-genesis.go

```diff
@@ -116,6 +117,7 @@ var CompressGenesisCmd = cli.Command{
 
 		// convert into allocation data
 		out := Genesis{
+			Config:        &superchain.GenesisConfig{Optimism: (*superchain.OptimismConfig)(genesis.Config.Optimism)},
```

Comment:
why the explicit typing?

Reply:
This is a bit of a gnarly line :-) Improved readability here [27d4033](https://github.com/ethereum-optimism/superchain-registry/pull/510/commits/27d4033e7924e8eaae815a1e025590262576a2a6).